fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/LdpContainerRdfContext.java

```diff
@@ -76,7 +74,10 @@ public LdpContainerRdfContext(final FedoraResource resource,
             throws RepositoryException {
         super(resource, idTranslator);
         final Iterator<Property> memberReferences = resource.getNode().getReferences(LDP_MEMBER_RESOURCE);
-        final Iterator<Property> properties = Iterators.filter(memberReferences, isContainer );
+        final Iterator<Property> properties = Iterators.filter(memberReferences, UncheckedPredicate.uncheck(p -> {
```

Comment:
No, that's `UncheckedFunction::uncheck`. Try statically importing it and you will see that the compiler gets confused and can't do the type inference.